docs/manifesto.md

```diff
@@ -0,0 +1,47 @@
+# Shaken Fist Manifesto
+
+This document attempts to list Shaken Fist's defining features, give guidance on what type of features should be added to the project, how they should be implemented and how we work together.
+
+## Shaken Fist Defining Characteristics
+
+* Shaken Fist is smaller, simpler cloud.
+* It is designed for relatively small environments with minimum management overhead.
+* It's features are highly opinionated. This means that the maintainers have chosen the best (in their opinion) features to support.
+* Opinionated features do not handle every single possible use case. This reduces the code base size thus increasing long-term maintainability.
+* The code base aims to be understandable in it's entirety by a single developer.
```

Comment:
```suggestion
* The code base is understandable in it's entirety by a single developer.
```
I don't see why this should ever be less than fact

Reply:
Changed. I like your optimism.

(Some days I don't understand the code, but I may not be a representative developer.)